src/modelfree/score_agent.py

```diff
@@ -48,7 +48,10 @@ def get_empirical_score(_run, env, agents, episodes, render=False):
 
 @score_agent_ex.named_config
```

Comment:
I'm not convinced we need `human_score_config` as a named config at all. Is the episodes being 20 specific to humans? If not, maybe just make it the default across the board?